couchbase/datadog_checks/couchbase/couchbase.py

```diff
@@ -237,26 +237,32 @@ def _create_metrics(self, data, tags=None):
         for key, storage_type in storage_totals.items():
             for metric_name, val in storage_type.items():
                 if val is not None:
-                    metric_name = '.'.join(['couchbase', key, self.camel_case_to_joined_lower(metric_name)])
+                    metric_name = '.'.join(
+                        ['couchbase', key, self.camel_case_to_joined_lower(metric_name)])
```

Comment:
To maintain readability, could you please put back any of these lines that didn't previously exceed 120 characters?